packages/util-linux/util-linux.spec

```diff
@@ -283,6 +286,8 @@ done
 %exclude %{_cross_sbindir}/vigr
 %exclude %{_cross_sbindir}/vipw
 %exclude %{_cross_sbindir}/zramctl
+%exclude %{_cross_sbindir}/*.debug
```

Comment:
Probably same explanation as the CNI plugins `exclude` statement for .debug files

Reply:
Deleted these files, it builds fine.